content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+```dart
+Alias name: edf-clustera-ca
+Creation date: Feb 16, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=MapR Engineering Signing CA, OU=MapR Engineering Signing CA, O=MapR, DC=hpecorp, DC=net
+Issuer: CN=MapR Engineering Root CA, OU=MapR Engineering Root CA, O=MapR, DC=hpecorp, DC=net
+...
+```
+
+#### 2. The documentation describes several client software that you can use to interact with the HPE Ezmeral Object Store. I would like to add the following clarification:
+
+When using the `mc` command-line tool or the `s3cmd` command-line tool to interact with HPE Ezmeral Data Fabric Object Store without completing the configuration of "[Enabling S3 Virtual-Host-Style Requests](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_nnp_kr2_bvb)", some commands will not work properly.
+
+In this case, for management operations, such as creating accounts, IAM accounts, buckets, etc., I recommend you use the `mc` command line tool.
+
+For object listing, getting, putting, and deleting operations, I recommend you use `AWS CLI`.
+
+#### 3. Cannot revert to HTTP mode after enabling HTTPS.
+
+If the Object Store was installed using the Installer, the Object Store would also have security enabled, including HTTPS.
+
+Although you can see the below description:
+
+> To revert to http access, comment out the moss.certs.dir=/opt/mapr/conf line in the /opt/mapr/conf/moss.conf file.
+
+from here - [HTTP Access to Object Store](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_bg1_zd1_vsb), but this configuration change alone does not change Object Store to HTTP mode, and there is nothing else in HPE Ezmeral Data Fabric's documentation on how to modify Object Store's TLS mode.
+
+### [](#create-a-bucket-in-hpe-ezmeral-data-fabric-object-store-and-upload-some-objects)Create a bucket in HPE Ezmeral Data Fabric Object Store and upload some objects
+
+In order to create a Bucket, you need to create an Account, an IAM User, and finally a Bucket in sequence.
+While creating IAM User and Bucket, you also need to prepare Access Policy to control who (IAM User) can perform which (list, get, put, etc.) operations on which Bucket.
+
+You can read the following document - [Entities and Resources](https://docs.datafabric.hpe.com/72/MapROverview/object_store_account_management.html) first to gain a deeper understanding of the entity model of HPE Ezmeral Data Fabric Object Store.
+
+Below, I will demonstrate in turn how to create the above-required entities.
+
+#### Create an account
+
+I choose to use the [mc](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-commands-overview.html) command line tool to create Account.
+
+Before using the mc command line for the first time, you need to create an Alias for your administrator.
```

Comment:
While I have seen "Alias" capitalized before, if we are using just sentence case, I would argue for "alias". If you think it needs to stand out more or be emphasized, please feel free to use a bold font.